plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -727,11 +730,12 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
       duration = animationTime
     }
 
-    bearingAnimator.addUpdateListener {
+    anchorAnimator.addUpdateListener {
```

Comment:
any reason removing the bearingAnimator's update listener.

Reply:
yes, we actually create bearing + anchor animator and run them as a set.
so it's more logical to reset anchor when anchor animator ends (although it's technically same moment when bearing animator ends).